pkg/cloudprovider/aws/suite_test.go

```diff
@@ -157,6 +160,26 @@ var _ = Describe("Allocation", func() {
 	})
 
 	Context("Reconciliation", func() {
+		Context("Standard Labels", func() {
+			It("should apply OS label based on the AMI Family", func() {
+				ExpectApplied(ctx, env.Client, provisioner)
```

Comment:
This was the issue Brandon saw, we've never labeled the nodes correctly but it didn't matter since we pre-bound and the kubelet would generate these labels anyway on startup.  With our inflight support we need to know the correct final labels.